### ISSUE TITLE
Display rollup stats for workspace on cargo test?

### ISSUE THREAD

**Issue by thomasmost** [rust-lang/cargo#11945]
### Problem

Is there any way to get `cargo test` to output the total number of tests run, ignored, failed, etc (let alone coverage stats) for an entire workspace after running tests for each member on `cargo test --` in a workspace? I have a fairly large workspace project with a dozen modules and it would be helpful to see these numbers aggregated

### Proposed Solution

with the `--rollup` flag passed, `cargo test` would output workspace stats at the end of the test run

```sh
  > Rollup: 243 tests run: 241 succeeded — 2 failed — 7 ignored
```

In the future this flag could also generate coverage reports, but at present I think this would be a simple ergonomic improvement for workspaces

**Comment by weihanglo** [comment]
Not exactly the same but #11532 also proposes to have a flag to output a summary for cargo commands. See #11542 for the discussion around that topic.

**Comment by weihanglo** [comment]
Feel like this is also blocked on libtest's JSON output. Cargo needs a reliable way to collect such data. See <https://github.com/rust-lang/cargo/issues/11875#issuecomment-1479603968> for more. In the meantime, personally I'd recommend [nextest](https://github.com/nextest-rs/nextest), which has a handful of goodies for testing user experience.

**Comment by thomasmost** [comment]
Thank you! I'll check out `nextest` — I appreciate it!

**Comment by weihanglo** [comment]
Realized today that this is a duplicate of #8414. I'll close this in favor of that. Thank you for the proposal.
